.rubocop.yml

```diff
@@ -319,6 +321,10 @@ Performance/ReverseEach:
 
 Performance/StringReplacement:
   Enabled: true
+  Exclude:
+    - 'activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb'
+    - 'activerecord/test/cases/validations_test.rb'
+    - 'activesupport/test/xml_mini_test.rb'
```

Comment:
This is what came out of Rubocop when I ran it:

```
activesupport/test/xml_mini_test.rb:356:36: C: [Correctable] Performance/StringReplacement: Use tr instead of gsub.
      assert_equal expected_base64.gsub(/\n/, " ").strip, parser.call(base64)
                                   ^^^^^^^^^^^^^^^
activesupport/test/xml_mini_test.rb:360:36: C: [Correctable] Performance/StringReplacement: Use tr instead of gsub.
      assert_equal expected_base64.gsub(/\n/, " ").strip, parser.call(base64, "encoding" => "base64")
                                   ^^^^^^^^^^^^^^^
```

Reply:
Not sure what is going on, but you haven't touched that file, and it passes CI. Anyway, I don't think you need any of these exclusion, the code can both be made more performant and not need any exception.

Reply:
Thanks, @byroot - I've removed the exclusions and merged the `gsub` calls as you suggested above.

Reply:
I've just added `if_exists: true` to the `remove_check_constraint` call, I'm assuming it's just a timing issue but hopefully this will help me figure out what's going wrong otherwise.